src/main/java/edu/kis/powp/jobs2d/command/Factories.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.powp.jobs2d.command;
+
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Factories {
+
+
+    public static List<DriverCommand> Square(){
```

Comment:
incorrect naming convention (methods start from a lowercase).
The return type should be IDriverCommand.